eq_cir_converter_service/services/schema/schema_processor_service.py

```diff
@@ -1,12 +1,44 @@
 """This module converts the schema from the current to the target version."""
 
+from collections.abc import Mapping, Sequence
+from typing import Any
+
+from jsonpath_ng.ext import parse
+
 from eq_cir_converter_service.config.logging_config import logging
+from eq_cir_converter_service.services.schema.extractable_strings import (
+    EXTRACTABLE_STRINGS,
+)
 from eq_cir_converter_service.types.custom_types import ConvertedSchema, InputSchema
+from eq_cir_converter_service.utils.helper_utils import process_element
 
 logger = logging.getLogger(__name__)
 
 
-async def convert_schema(current_version: str, target_version: str, schema: InputSchema) -> ConvertedSchema:
+# --- JSONPath-Based Transformation ---
+def transform_json_schema(schema: Mapping[str, Any], paths: Sequence[Mapping[str, str]]) -> Mapping[str, Any]:
+    """Transforms the JSON schema based on the provided JSONPath expressions."""
+    for path_expr in paths:
+        if isinstance(path_expr, dict) and "json_path" in path_expr:
+            expr = parse(path_expr["json_path"])
```

Comment:
Can we eliminate some more extractable strings? Possibly if we ask the team responsible for CIR or someone senior (eg. "playback", is playback even used?. Is "playback" not census specific? It was added for relationship questions, are relationship questions still present in business, social, health? This would, again, make the logic a lot easier the more scenarios we eliminate.